src/theme-manager/themeReducer.ts

```diff
@@ -68,5 +68,7 @@ const {
   },
 })
 
-export { customThemeKey, themeActions }
+const { updateGlobalTheme, updateThemeComponent, updateTheme, clearTheme } = themeActions
+
+export { updateGlobalTheme, updateThemeComponent, updateTheme, clearTheme }
```

Comment:
I was thinking that we can export these actions so that the app consuming this plugin can use them to add their custom theme. There's a helper fn in `/theme-manager/helpers` that helps them create the theme definition which can then be passed into the `updateTheme` action here.

Not sure if this is the correct way to do it. Also, wouldn't there be the issue where on the first render, the theme would be incorrect?